src/shared/afunix.rs

```diff
@@ -0,0 +1,10 @@
+use shared::minwindef::DWORD;
```

Comment:
nice! I came up with practically the same patch without knowing you add a MR.

I think you should add the common top header:
```
// Licensed under the Apache License, Version 2.0
// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
// All files in the project carrying such notice may not be copied, modified, or distributed
// except according to those terms.
//! Definitions for the AF_UNIX socket address family.
```